includes/class-wc-payments-account.php

```diff
@@ -1537,10 +1537,11 @@ public function maybe_handle_onboarding() {
 					$create_test_drive_account ? 'test_drive' : ( $should_onboard_in_test_mode ? 'test' : 'live' ),
 					$wcpay_connect_param,
 					[
-						'promo'       => ! empty( $incentive_id ) ? $incentive_id : false,
-						'progressive' => $progressive ? 'true' : false,
-						'source'      => $onboarding_source,
-						'from'        => WC_Payments_Onboarding_Service::FROM_STRIPE,
+						'promo'                       => ! empty( $incentive_id ) ? $incentive_id : false,
+						'progressive'                 => $progressive ? 'true' : false,
+						'collect_payout_requirements' => $collect_payout_requirements ? 'true' : false,
```

Comment:
I suggest to improve consistency a bit - either make both `'true'` and `false` strings or booleans

Reply:
@oaratovskyi No no. They need to be this way if we want for those params to not be present in the URL (boolean false ones will not be included in the URL).

Reply:
gotcha!